inte-pdp-components-openmedia.yaml

```
@@ -181,6 +177,19 @@ components:
           example: "legend"
           description: "the legend of the image"
 
+    RokkaioImageUrl:
```

Damit es konsistent mit dem Naming von `RokkaIoImage` ist:
```suggestion
    RokkaIoImageUrl:
```